bucket/cygwin.json

```diff
@@ -31,6 +31,14 @@
             "root\\bin\\bash.exe",
             "cygwin",
             "--login -i"
+        ],
+        [
+            "setup-x86_64.exe",
+            "Cygwin-Setup"
```

Comment:
Thanks for review! I tried autocomplete and found that suggestions sort by name in ascending order, so it would not interfere you. But I think your naming scheme is better, cause it follows Verb-Noun naming rule.

I have added `Setup Cygwin` shortcut, but still retained `Setup-Cygwin` shim in `bin` to be able to use by CLI.